card/mod/03_machines/set/type/css.rb

```diff
@@ -29,7 +29,7 @@ def chunk_list  #turn off autodetection of uri's
 
 
 format :html do
-  def get_inclusion_defaults nested_card
+  def get_nest_defaults nested_card
```

Comment:
Unused method argument - `nested_card`. If it's necessary, use `_` or `_nested_card` as an argument name to indicate that it won't be used. You can also write as `get_nest_defaults(*)` if you want the method to accept any arguments but don't care about them.